packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -31,6 +32,29 @@ const Link = ({ to, onClick, ...props }) => {
     }
   `);
 
+  const properties =
+    typeof instrumentation === 'string'
+      ? { customProp: instrumentation }
+      : instrumentation;
+
+  const handleClick = useInstrumentedHandler(onClick, {
```

Comment:
[nit] This used to be called `handleExternalLinkClick` to clearly call out that this handler was meant for the external link case only. Could you rename to be the same?

Reply:
Oops yes, sorry I was just pasting in from when I had it namespaced to ExternalLink! it has been changed back